src/drt/src/dr/FlexDR_maze.cpp

```diff
@@ -1559,7 +1559,9 @@ void FlexDRWorker::route_queue()
     setMarkers(gcWorker_->getMarkers());
   }
   if (getDRIter() >= beginDebugIter) {
-    cout << "Starting worker " << getRouteBox() << " with " << markers_.size() << " markers\n";
+    stringstream ss;
+    ss << "Starting worker " << getRouteBox() << " with " << markers_.size() << " markers\n";
+    logger_->info(DRT, 2001, ss.str());
```

Comment:
why are you using a stringstream?  Just 
logger_->info(DRT, 2001, "Starting worker {} with {} markers", getRouteBox(),  markers_.size());

Reply:
@maliberty I think it's because `getRouteBox()` returns a `Rect` which only has a `dbStream` friend function but not `iostream`. spdlog will complain that `Rect` is not formattable, but if it gets put into a stringstream first then it works. This will be a lot easier if/when dbStreams are removed.